### PR TITLE
Codewind remote changes for #3168

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <MICGIBSO@uk.ibm.com>

## What type of PR is this ? 

- [ ] Bug fix
- [X] Enhancement

## What does this PR do ?
Updates the help with the recent changes to the codewind-operator. 

## Which issue(s) does this PR fix ?
https://github.com/eclipse/codewind/issues/3168

## Does this PR require a documentation change ?
Yes.

## Any special notes for your reviewer ?
Checked with Acrolinx. 
Also changed the `$ kubectl get codewind -n codewind` to `$ kubectl get codewinds -n codewind` to match the codewind-operator readme.

### CODE REVIEW

**Comment by micgibso** [comment]
@markcor11 Please review, thx.

File: docs/_documentations/remote-deploying-codewind.md [hunk 1]
```diff
@@ -164,15 +164,27 @@ Your Codewind instance is deployed. Access it by way of the **Access URL** retur
...
 This command returns output in the format of the following example:
 
 ```
-NAME       USERNAME   NAMESPACE   AGE    KEYCLOAK   REGISTRATION   ACCESSURL
-devex001   jane       codewind    119m   devex001   Complete       https://codewind-gatekeeper-devex001.<ingress-domain>
+NAME    USERNAME   NAMESPACE   WORKSPACE      AGE    KEYCLOAK   REGISTRATION   ACCESSURL
+jane1   jane       codewind    kbc3b0x2qins   119m   devex001   Complete       https://codewind-gatekeeper-kbc3b0x2qins.codewind.<ingress-domain>
 ```
 
+For the 0.14.0 release of Codewind, a new `WORKSPACE` field appears in the output. This field is a randomly generated set of characters that are used in both the URL of the Gatekeeper and the name of the pods and resources that you create.
```
Comment (markcor11) [inline docs/_documentations/remote-deploying-codewind.md:176]: The user doesn't create or use the WORKSPACE field other than to use it for identification. Reading this sentence it suggests this value will appear in PROJECT urls which it does not. I think we can just snip off "that you create"

File: docs/_documentations/remote-deploying-codewind.md [hunk 2]
```diff
@@ -164,15 +164,27 @@ Your Codewind instance is deployed. Access it by way of the **Access URL** retur
...
-devex001   jane       codewind    119m   devex001   Complete       https://codewind-gatekeeper-devex001.<ingress-domain>
+NAME    USERNAME   NAMESPACE   WORKSPACE      AGE    KEYCLOAK   REGISTRATION   ACCESSURL
+jane1   jane       codewind    kbc3b0x2qins   119m   devex001   Complete       https://codewind-gatekeeper-kbc3b0x2qins.codewind.<ingress-domain>
 ```
 
+For the 0.14.0 release of Codewind, a new `WORKSPACE` field appears in the output. This field is a randomly generated set of characters that are used in both the URL of the Gatekeeper and the name of the pods and resources that you create. 
+
+To help you to determine the unique resources for your deployment, you can use the `NAME` field or you can use the new workspace ID given in the `WORKSPACE` field. All of your Codewind deployments have both the `NAME` and `WORKSPACE` as labels for you to use. 
+
+Also, new for 0.14.0, the namespace appears in the `ACCESSURL`. The namespace label is to enable the Codewind Operator to deploy into different namespaces. In the example, the namespace is `codewind` and it appears just before `<ingress-domain>`.
```
Comment (markcor11) [inline docs/_documentations/remote-deploying-codewind.md:180]: We don't yet support the operator running in different namespaces.  Right now we insist the namespace must be "codewind". This change of adding the namespace is part of our journey towards allowing different namespaces but we're not finished yet.

File: docs/_documentations/remote-deploying-codewind.md [hunk 3]
```diff
@@ -164,15 +164,27 @@ Your Codewind instance is deployed. Access it by way of the **Access URL** retur
...
+
+To help you to determine the unique resources for your deployment, you can use the `NAME` field or you can use the new workspace ID given in the `WORKSPACE` field. All of your Codewind deployments have both the `NAME` and `WORKSPACE` as labels for you to use. 
+
+Also, new for 0.14.0, the namespace appears in the `ACCESSURL`. The namespace label is to enable the Codewind Operator to deploy into different namespaces. In the example, the namespace is `codewind` and it appears just before `<ingress-domain>`. 
+
+If you deployed remote Codewind instances before the Codewind 0.14.0 release, you must regenerate the connections to the deployments:
+1. Delete your deployments. **Important:** You must do this action before you delete the Codewind namespace. If you do not, your Codewind deployments might go into pending state and not be removed. If you delete the Codewind namespace before you delete your deployments, you can run the following command to clear them manually: `$ kubectl patch codewind {codewindName}  -p '{"metadata":{"finalizers": []}}' --type=merge`.
```
Comment (markcor11) [inline docs/_documentations/remote-deploying-codewind.md:183]: Delete your deployments -> "Delete all the Codewind deployments in the codewind namespace"

File: docs/_documentations/remote-deploying-codewind.md (hunk 1 above)
Comment (micgibso) [reply docs/_documentations/remote-deploying-codewind.md:176]: Corrected.

File: docs/_documentations/remote-deploying-codewind.md (hunk 2 above)
Comment (micgibso) [reply docs/_documentations/remote-deploying-codewind.md:180]: Changed the sentence to read "Also, new for 0.14.0, the `codewind` namespace appears in the `ACCESSURL`. In the example, it appears just before `<ingress-domain>`. "

File: docs/_documentations/remote-deploying-codewind.md (hunk 3 above)
Comment (micgibso) [reply docs/_documentations/remote-deploying-codewind.md:183]: Amended.

**Comment by micgibso** [comment]
Broken link fixed by https://github.com/eclipse/codewind-docs/pull/680

**Pull request merged**